Dockerfile

```
@@ -0,0 +1,15 @@
+FROM opensciencegrid/software-base:3.6-el7-release
+
+LABEL maintainer OSG Software <help@opensciencegrid.org>
+
+COPY . /src
+
+RUN yum update -y && \
+    yum install -y python3 && \
+    yum install -y python36-m2crypto.x86_64 && \
```

```suggestion
    yum install -y python36-m2crypto && \
```

You can specify multiple packages in the arguments to `yum install` but we shouldn't need to in this case since `python36-m2crypto` will require `python3` and install it as one of its dependencies. We also don't need to specify the architecture (`x86_64`)